ssh/tailssh/incubator_darwin.go

```diff
@@ -19,3 +29,15 @@ func setGroups(groupIDs []int) error {
 	// this to work for more things than it previously did.
 	return syscall.Setgroups(groupIDs[:16])
 }
+
+// maybeRunCommandWithLoginDarwin is the darwin implementation of maybeRunCommandWithLogin
+func maybeRunCommandWithLoginDarwin(logf logger.Logf, ia incubatorArgs) (bool, error) {
+	if ia.isSFTP || ia.isShell || ia.cmdName == "" || ia.loginCmdPath == "" {
+		return false, nil
+	}
+
+	// if we run it this way on Mac, this seems to solve various user ID issues
```

Comment:
This comment makes sense to you today but will make sense to ~nobody in a month or two.

Either say more or include a URL to more info.

Reply:
Ack, done.